WooCommerce/src/main/kotlin/com/woocommerce/android/model/GetLocations.kt

```diff
@@ -0,0 +1,22 @@
+package com.woocommerce.android.model
+
+import com.woocommerce.android.ui.orders.details.editing.address.LocationCode
+import org.wordpress.android.fluxc.store.WCDataStore
+import javax.inject.Inject
+
+class GetLocations @Inject constructor(private val locationStore: WCDataStore) {
```

Comment:
```suggestion
class GetLocations @Inject constructor(private val locationRepository: LocationRepository) {
```

ideally, it should be `LocationRepository` which would provide `Location` app models instead of `WCLocation` database model, similarly to e.g. `OrderDetailRepository`. This way we wouldn't have to call mapping methods in this quasi-use case class. 

But to scope this work reasonably, I've decided to not add this in this PRs but postpone it (to moment where `WCDataStore` will operate on Room models).